doc/Building_FRR_on_Ubuntu1204.md

```diff
@@ -94,7 +94,6 @@ an example.)
         --enable-rtadv \
         --enable-tcp-zebra \
         --enable-fpm \
-        --enable-ldpd \
```

Comment:
Do we need to change debian/rules or the redhat/frr.spec files.  Non-blocking comment.